### ISSUE TITLE
Permissions Denied

### ISSUE THREAD

**Issue by khisanthax** [T9Air/Klipper_Power_Resume#70]
I wanted to add, that I reinstalled the new version, and I do like that it now tells me there's an updated version that I can install. However, I'm still having the same issue that I have to chown the directory, Klipper Power Resume, to my user otherwise no operations are permitted in that directory and nothing is created.  I'm running Ubuntu 22.04 and I believe root is disabled if that matters.

**Comment by T9Air** [comment]
@khisanthax The ownership should be fixed by the install script automatically in v3.5.2.

**Comment by khisanthax** [comment]
I think when I went to 3.6.0, it said there was the latest 3.5.0 and asked if I wanted to upgrade and said yes, which actually downgraded me.  So I'll do that again and maybe it'll resolve my macro issue.

**Comment by T9Air** [comment]
Do not update to 3.5.0, it does not have macros enabled. The reason it downgrades from 3.6.0 to 3.5.0 is because 3.5 is the latest release, while 3.6 is currently in development. I replied on the other issue with how to make the macro work.

**Comment by khisanthax** [comment]
I removed the KPR directory and ran the git command for the 3.6.0 branch.  I I went to the directory, bash kpr.sh and pressed one for install but I could operation denied still.  I chown'd Klipper_Power_Resume but still received some errors as shown below:

https://i.imgur.com/TD3yDs6.png

**Comment by T9Air** [comment]
I believe that I fixed it in v3.5.2, but I forgot to merge those changes into the v3.6.0 branch. It should work now.

**Comment by khisanthax** [comment]
thanks, I'll try it again after this batch of printing!

**Comment by T9Air** [comment]
@khisanthax It works?

**Comment by T9Air** [comment]
I am assuming it works, so I will close the issue.

**Comment by khisanthax** [comment]
I'm sorry for not getting back.  I tried it again, removed the directory and downloaded 3.6.0 but still had the same issue with permissions.

**Comment by khisanthax** [comment]
Should I try the 3.5.2 branch to see if it's fixed there for me?

**Comment by T9Air** [comment]
I merged 3.5.2 into the main branch, so you could try installing with the regular instructions

**Comment by khisanthax** [comment]
I'll try that and report back

**Comment by khisanthax** [comment]
It didn't seem to work.  The first two images were before I tried to chown the directory as the main user, the third image was what happened after I chown'd it.

https://i.imgur.com/hG5sYTF.png
https://i.imgur.com/B6hbMJn.png
https://i.imgur.com/A384PM6.png

**Comment by T9Air** [comment]
It should be fixed now, it seems that the paths to the files were messed up in the install script. If you can confirm that it works, it would be greate

**Comment by khisanthax** [comment]
I had to chown the directory that git created, but after that it installed
fine.  I used the main branch, and I saw now that the add logs button
allows me to type in the name of the file.  I didn't see a macro for
resuming.  Would it be possible to add something like kpr in front of your
macro's so they're grouped together?

**Comment by T9Air** [comment]
Meaning in front of the name of the macro, right?

**Comment by khisanthax** [comment]
Yeah, I noticed that klipper while sort by first letter, so they'll all be
together if it starts with the same acronym.  Just a silly idea.

**Comment by T9Air** [comment]
@khisanthax I added the resume print macro. I am not sure how well it works, as I am currently printing something, but if you want to try it out know, it should work.

**Comment by khisanthax** [comment]
That sounds great!  I'm in the middle of some prints now but if so get a chance before I go on vacay I will update and give it a shot!

**Comment by T9Air** [comment]
@khisanthax If you have any updates, it would be great to know if it works. I would prefer if you could give feedback on the pull request. #68

**Comment by khisanthax** [comment]
I will try this again tomorrow and give feedback, thanks for remembering!

**Comment by khisanthax** [comment]
Okay, the install worked beautifully, no problems at all!  After I added the printer I got this error: https://i.imgur.com/OwsLXGx.png.  I tried to uninstall but it didn't remove the affected code, presumably because it didn't remove the additional change in macro so I just had to delete those lines manually.  And the uninstall gave me this: https://i.imgur.com/Gg5cOeN.png, but I don't think that matters really.
